packages/core/src/State.ts

```diff
@@ -18,6 +19,17 @@ type GraphProperties = {
   components?: Rule[][]
 }
 
+function getConfigHome (): string {
+  switch (process.platform) {
+    case 'win32':
+      return process.env.APPDATA || path.join(os.homedir(), 'AppData', 'Roaming')
+    case 'darwin':
+      return path.join(os.homedir(), 'Library', 'Application Support')
```

Comment:
I understand that according to the Apple guideline app-specific configurations should be placed under `~/Library/Application Suport`. That being said, since macOS is also a UNIX, some users want to put them in `$XDG_CONFIG_HOME`.  If we respect `$XDG_CONFIG_HOME` in case of `process.platform == "darwin"`, `~/Library/Application Suport` can be used if `$XDG_CONFIG_HOME` is _not_ defined.

I have to admit I'm saying something double standard, but what's your idea?

Reply:
I think that sounds fine, since `$XDG_CONFIG_HOME` is extremely unlikely to be set on MacOS normally.